src/Fields.php

```diff
@@ -573,6 +573,9 @@ protected function wf_crm_get_fields($var = 'fields') {
         // Fetch case roles
         $sets['caseRoles'] = ['entity_type' => 'case', 'label' => t('Case Roles')];
         foreach ($this->utils->wf_crm_apivalues('case_type', 'get') as $case_type) {
+          if (!$case_type['is_active'] || !isset($case_type['definition']['caseRoles'])) {
+            continue;
+          }
           foreach ($case_type['definition']['caseRoles'] as $role) {
```

Comment:
@herbdool Can we check for active condition in the API instead? and maybe, optimise the above changes to the following?

````
foreach ($this->utils->wf_crm_apivalues('case_type', 'get', ['is_active' => 1]) as $case_type) {
  foreach ($case_type['definition']['caseRoles'] ?? [] as $role) {
````

Reply:
I've changed the PR with your suggestion @jitendrapurohit